docs/README.rst

```diff
@@ -9,10 +9,10 @@ This file provides a quick guide on how to compile the Scrapy documentation.
 
 Setup the environment
 ---------------------
+.. highlight:: bash
 
 To compile the documentation you need Sphinx Python library. To install it
 and all its dependencies run the following command from this dir
-
 ::
```

Comment:
You should either leave this as it was or, better yet, put `::` at the end of the preceding paragraph.